submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment.component.ts

```diff
@@ -101,7 +101,7 @@ export class ExperimentComponent implements OnInit {
       this.modalProps = { ...this.modalProps, ...props };
     });
 
-    this.reloadCheck();
+    // this.reloadCheck();
```

Comment:
Why comment out this line?

Reply:
Oh, I accidentally commented on this line. 
However, I think there is a better way to handle this issue. I will check it later when I start to refactor the code.